src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1092,7 +1092,7 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
                 if (setMethod is null)
                 {
                     var containing = this.ContainingMemberOrLambda;
-                    if (!AccessingAutoPropertyFromConstructor(receiver, propertySymbol, containing)
+                    if (!IsPropertyAssignableFromConstructor(receiver, propertySymbol, containing)
```

Comment:
>IsPropertyAssignableFromConstructor [](http://example.com/codeflow?start=25&length=35)

It feels like this is the only place where a name like "IsPropertyAssignable" would be a suitable name for the used helper. However, "IsPropertyAssignableThrougBackingField" feels more specific.